.circleci/continue_config.yml

```diff
@@ -0,0 +1,771 @@
+            - packages/components/pages/f-loyalty/dist
+            - packages/components/pages/f-offers/dist
+            - packages/components/pages/f-registration/dist
+            - packages/components/pages/f-takeawaypay-activation/dist
+            - packages/components/pages/f-account-info/dist
+
+      - save_cache:
+          name: Cache for services
+          key: build-cache-services-{{ .Branch }}-{{ .Revision }}
+          paths:
+            - packages/services/f-analytics/dist
+            - packages/services/f-braze-adapter/dist
+            - packages/services/f-compatibility/dist
+            - packages/services/f-consumer-oidc/dist
+            - packages/services/f-development-context/dist
+            - packages/services/f-feature-management/dist
+            - packages/services/f-globalisation/dist
+            - packages/services/f-http/dist
+            - packages/services/f-services/dist
+            - packages/services/f-performance/dist
+            - packages/services/f-statistics/dist
+
+  restore_cache_dist_directories:
+    description: Restores all the dist dircetories
+    steps:
+      - restore_cache:
+          name: Restore molecules Cache
+          keys:
+            - build-cache-molecules-master-
+      - restore_cache:
+          name: Restore atoms Cache
+          keys:
+            - build-cache-atoms-master-
+      - restore_cache:
+          name: Restore organisms Cache
+          keys:
+            - build-cache-organisms-master-
+      - restore_cache:
+          name: Restore pages Cache
+          keys:
+            - build-cache-templates-master-
+      - restore_cache:
+          name: Restore pages Cache
+          keys:
+            - build-cache-pages-master-
+      - restore_cache:
+          name: Restore services Cache
+          keys:
+            - build-cache-services-master-
+
+  build_all_components:
+    description: Builds all components in the monorepo
+    steps:
+      - run_command:
+          command_description: Build all components
+          command_name: build
+
+  build_component_type:
+    description: Builds a component type in the monorepo
+    parameters:
+      component_type:
+        type: string
+        default: global
+    steps:
+      - run_command:
+          command_description: Build << parameters.component_type >> components
+          command_name: ci:build:<< parameters.component_type >>
+
+  lint_all_components:
+    description: Lints all components in the monorepo
+    steps:
+      - run_command:
+          command_description: Lint all components
+          command_name: lint
+
+  lint_component_type:
+    description: Lints << parameters.component_type >> components
+    parameters:
+      component_type:
+        type: string
+        default: global
+    steps:
+      - run_command:
+          command_description: Run linting for << parameters.component_type >> components
+          command_name: ci:lint:<< parameters.component_type >>
```

Comment:
Same as above, but for linting.

We can probably condense a lot of this duplicate logic, but this PR was mainly focused on just getting the new workflow working.